moralization/analyse.py

```diff
@@ -71,14 +73,55 @@ def sort_spans(cas: object, ts: object) -> defaultdict:
     return span_dict
 
 
-def get_sentences(cas: object, ts: object) -> list:
+def add_true_sentences(cas: object, ts: object) -> object:
+    nlp = spacy.load("de_core_news_sm")
+    text = cas.sofa_string
+    sentences = [sentence for sentence in nlp(text).sents]
+    # create new typesystem:
+    try:
+        True_sentence_type = ts.create_type(name="moralization.TrueSentence")
+    except ValueError:
+        True_sentence_type = ts.get_type("moralization.TrueSentence")
+
+    for sentence in sentences:
+        if (sentence.end - sentence.start) > 3:
```

Comment:
could `3` be a variable with a name that suggests something about what it is, e.g. `min_sentence_length`?

Reply:
done under the name `minimum_tokens_per_sentence`